app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
+    @review.user_id = current_user.id
+    @review.video_id = params[:video_id]
+    @review.rating = params[:Rating]
```

Comment:
Why are you resetting rating here? I see you have it in params_review as permitted attribute so basically this looks a little bit over. Also why `R` on rating?